components/status_dropdown/index.jsx

```diff
@@ -21,11 +22,13 @@ function mapStateToProps(state) {
         return {};
     }
 
+    const userId = currentUser.id;
     return {
-        currentUser,
-        profilePicture: Client4.getProfilePictureUrl(currentUser.id, currentUser.last_picture_update),
-        autoResetPref: get(state, Preferences.CATEGORY_AUTO_RESET_MANUAL_STATUS, currentUser.id, ''),
-        status: getStatusForUserId(state, currentUser.id),
+        userId,
+        profilePicture: Client4.getProfilePictureUrl(userId, currentUser.last_picture_update),
+        autoResetPref: get(state, Preferences.CATEGORY_AUTO_RESET_MANUAL_STATUS, userId, ''),
+        status: getStatusForUserId(state, userId),
+        customStatus: getCustomStatus(state),
```

Comment:
Should we change this selector to pass an ID?
This way, we can get the status for all users.

We can default to the current user's status.